iot/mqtt_example/cloudiot_mqtt_example_nodejs.js

```diff
@@ -282,10 +287,15 @@ client.on('error', err => {
 });
 
 client.on('message', (topic, message) => {
-  console.log(
-    'message received: ',
-    Buffer.from(message, 'base64').toString('ascii')
-  );
+  let messageStr = 'Message received: ';
+  if (topic === `/devices/${argv.deviceId}/config`) {
+    messageStr = 'Config message received: ';
+  } else if (topic === `/devices/${argv.deviceId}/commands`) {
```

Comment:
It may be worth verifying the topic name fully matches what you expect here as we're subscribing with the wild card above.

Reply:
I actually tested this and using # doesn't actually match here